### PR TITLE
Add sponsorship page component

### LINKED ISSUE #12: Create homepage sponsors section
Screen Shot 2022-07-31 at 12 08 44 PM

### PR DESCRIPTION
**Pull request by tylerlam1**
Resolves #12.

**Web:**
image

**Mobile:**
image

### CODE REVIEW

File: components/Sponsors.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import Image from "next/image";
+import Heading from "./Heading";
+
+const SPONSORS_CONTAINER = "bg-white flex flex-col items-center";
```
Comment (harsohailB) [inline components/Sponsors.jsx]: Can we add `py-20` for some vertical spacing?

File: components/Sponsors.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import Image from "next/image";
+import Heading from "./Heading";
+
+const SPONSORS_CONTAINER = "bg-white flex flex-col items-center";
+const HEADING_CONTAINER = "flex flex-row justify-center mb-10";
+const SPONSORS_LEVEL_CONTAINER = "flex flex-row justify-center items-center my-5";
+const CONTENT_CONTAINER = "my-10 font-medium max-w-md text-center";
```
Comment (harsohailB) [inline components/Sponsors.jsx]: This container looks wider in figma. Maybe `font-base w-2/3`?

File: components/Sponsors.jsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  width: 177,
+  height: 10,
+};
+
+const Sponsors = () => {
+  // Multiplier is the value you need to multiply height by to get width
+  const goldSponsorsLevelOne = [
+    { file: "Benevity.png", multiplier: 3.216, height: 90 },
+    { file: "Hunter Hub.png", multiplier: 7.02, height: 70 },
+  ];
```
Comment (harsohailB) [inline components/Sponsors.jsx]: Can we take this out of the component and make a constant as well?

Comment (harsohailB) [reply components/Sponsors.jsx]: nit: redirect to the sponsor's website on click?

**Review by harsohailB — CHANGES_REQUESTED** [review]
Very nice! Gave some suggestions.

File: components/Sponsors.jsx (hunk 1 above)
Comment (tylerlam1) [reply components/Sponsors.jsx]: sounds g

File: components/Sponsors.jsx (hunk 2 above)
Comment (tylerlam1) [reply components/Sponsors.jsx]: Screen Shot 2022-08-08 at 3 32 37 PM
That's too wide on my screen. I'll play around with it a bit

File: components/Sponsors.jsx (hunk 3 above)
Comment (tylerlam1) [reply components/Sponsors.jsx]: 1. Will do
2. Will do

File: pages/_app.js [hunk 4]
```diff
@@ -13,9 +13,7 @@ function MyApp({ Component, pageProps }) {
       <Navbar />
       <Component {...pageProps} />
       {/* CONTAINER FOR CONTENT BELOW TO TEST NAVBAR IS IN RIGHT POSITION*/}
-      <div className="mb-auto bg-green-500">
-        <h1>content</h1>
-      </div>
+      <div className="mb-auto bg-green-500"></div>
```
Comment (harsohailB) [inline pages/_app.js]: I think you can just get rid of this now as we have content on master

File: components/Sponsors.jsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+const HEADING_CONTAINER = "flex flex-row justify-center mb-10";
+const SPONSORS_LEVEL_CONTAINER = "flex flex-row justify-center items-center my-5";
+const CONTENT_CONTAINER = "my-10 font-medium w-1/2 text-center ";
+const UNDERLINE_CONTAINER = "mb-10";
+const UNDERLINE_PROPERTIES = {
+  width: 177,
+  height: 10,
+};
+
+const Sponsors = () => {
```
Comment (harsohailB) [inline components/Sponsors.jsx:16]: I don't see anything rendered. Did you drop the diff on index.js? Place `<Sponsors />` in `index.js` for it to render

**Review by harsohailB — CHANGES_REQUESTED** [review]

File: pages/_app.js (hunk 4 above)
Comment (tylerlam1) [reply pages/_app.js]: Will do

File: components/Sponsors.jsx (hunk 5 above)
Comment (tylerlam1) [reply components/Sponsors.jsx:16]: Done

**Review by harsohailB — APPROVED** [review]
🔥

File: data/sponsors.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+// Multiplier is the value you need to multiply height by to get width
+export const GOLD_SPONSORS_ONE = [
+  { file: "Benevity.png", multiplier: 3.216, height: 90, website: "benevity.com" },
+  { file: "Hunter Hub.png", multiplier: 7.02, height: 70, website: "ucalgary.ca/hunter-hub" },
+];
+
+export const GOLD_SPONSORS_TWO = [
+  { file: "Arcurve.png", multiplier: 10, height: 50, website: "arcurve.com" },
+  { file: "Cisco Secure.jpg", multiplier: 3.41, height: 120, website: "cisco.com/site/us/en/products/security" },
```
Comment (topanb11) [inline data/sponsors.js]: I tried going onto the cisco site but it doesn't work, maybe put in another link?

File: components/Sponsors.jsx [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+      <a className="mx-5">
+        <Image src={`/sponsor_logos/${file}`} alt={file} width={height * multiplier} height={height} />
+      </a>
+    </Link>
+  );
+
+  return (
+    <div className={SPONSORS_CONTAINER}>
+      <div className={HEADING_CONTAINER}>
+        <Heading>Sponsors</Heading>
```
Comment (topanb11) [inline components/Sponsors.jsx:28]: On mobile, the sponsors title is slightly shifted left, could we align it a bit more right/center?

**Review by topanb11 — CHANGES_REQUESTED** [review]

File: data/sponsors.js (hunk 6 above)
Comment (tylerlam1) [reply data/sponsors.js]: Yeah not sure why it didn't work either - will just redirect to Cisco website

File: components/Sponsors.jsx (hunk 7 above)
Comment (tylerlam1) [reply components/Sponsors.jsx:28]: Will do

Comment (tylerlam1) [reply components/Sponsors.jsx:28]: Check it now

Comment (topanb11) [reply components/Sponsors.jsx:28]: Looks good bro

**Review by topanb11 — APPROVED** [review]
fire

**Pull request merged**